--- old_vagrant_init.py	2015-02-22 22:38:21.000000000 -0800
+++ new_vagrant_init.py	2015-02-22 22:39:03.000000000 -0800
@@ -705,16 +705,13 @@
         '''
         # Make subprocess command
         command = self._make_vagrant_command(args)
-        if self.quiet_stdout or self.quiet_stderr:
-            # Redirect stdout and/or stderr to devnull
-            # Use with stmt to close filehandle in case of exception
-            with open(os.devnull, 'wb') as fh:
-                outfh = fh if self.quiet_stdout else sys.stdout
-                errfh = fh if self.quiet_stderr else sys.stderr
-                subprocess.check_call(command, cwd=self.root,
-                                      stdout=outfh, stderr=errfh)
-        else:
-            subprocess.check_call(command, cwd=self.root)
+        # Redirect stdout and/or stderr to devnull
+        # Use with stmt to close filehandle in case of exception
+        with open(os.devnull, 'wb') as fh:
+            outfh = fh if self.quiet_stdout else sys.stdout
+            errfh = fh if self.quiet_stderr else sys.stderr
+            subprocess.check_call(command, cwd=self.root,
+                                  stdout=outfh, stderr=errfh)
 
     def _run_vagrant_command(self, args):
         '''
