--- vagrant_orig.py	2015-02-15 23:32:03.000000000 -0800
+++ vagrant_new.py	2015-02-15 23:32:37.000000000 -0800
@@ -20,7 +20,7 @@
 
 # python package version
 # should match r"^__version__ = '(?P<version>[^']+)'$" for setup.py
-__version__ = '0.5.2'
+__version__ = '0.5.3'
 
 
 log = logging.getLogger(__name__)
@@ -705,16 +705,13 @@
         '''
         # Make subprocess command
         command = self._make_vagrant_command(args)
-        if self.quiet_stdout or self.quiet_stderr:
-            # Redirect stdout and/or stderr to devnull
-            # Use with stmt to close filehandle in case of exception
-            with open(os.devnull, 'wb') as fh:
-                outfh = fh if self.quiet_stdout else sys.stdout
-                errfh = fh if self.quiet_stderr else sys.stderr
-                subprocess.check_call(command, cwd=self.root,
-                                      stdout=outfh, stderr=errfh)
-        else:
-            subprocess.check_call(command, cwd=self.root)
+        # Redirect stdout and/or stderr to devnull
+        # Use with stmt to close filehandle in case of exception
+        with open(os.devnull, 'wb') as fh:
+            outfh = fh if self.quiet_stdout else sys.stdout
+            errfh = fh if self.quiet_stderr else sys.stderr
+            subprocess.check_call(command, cwd=self.root,
+                                  stdout=outfh, stderr=errfh)
 
     def _run_vagrant_command(self, args):
         '''
